application/src/sass/_overrides.scss

```diff
@@ -13,3 +13,12 @@ Naming convention follows https://design-system.service.gov.uk/styles/spacing/
 .eff-\!-padding-bottom-40 {
   padding-bottom: 40px;
 }
+
+/* Overrides for specific colours */
+.background-light-blue {
```

Comment:
I guess we aren't really using [BEM](http://getbem.com/) (which GOV.UK Design System is based around) at the moment, but even so it might be nice to have modifiers separated by double hyphens rather than just singles. Makes it a bit more apparent it's an adjustment to the `background` class, rather than being its own special thing all of itself. Maybe?

Reply:
Or if we don't use BEM - do we have anything already in place around consistency for CSS styling selectors?

Reply:
Yeah, we don't have any consistency yet really. However I'm trying to move us towards the conventions from Design System.

I think these classes are more like the overrides in https://design-system.service.gov.uk/styles/typography/#font-override-classes which just use hyphens - as they're not really modifiers to a component or anything?